problems/arrayMethods.js

```diff
@@ -91,15 +144,31 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
-
+const counterObject = (items) => {
+  let newObj = {};
+  items.map((item) => {
```

Comment:
This should not be done with a map. Map is for returning a new array. You're using it for side effects. It should be a forEach.